oboe/hello-oboe/src/main/java/com/google/sample/oboe/hellooboe/MainActivity.java

```diff
@@ -74,6 +76,26 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
 
+        mAudioApiSpinner = findViewById(R.id.audioApisSpinner);
+        mAudioApiSpinner.setAdapter(new SimpleAdapter(
+                this,
+                createAudioApisOptionsList(),
+                R.layout.audio_apis_spinner, // the xml layout
+                new String[]{getString(R.string.audio_api_description_key)}, // field to display
```

Comment:
nit: `new int[] {5, 6} and new int[] { 5, 6 } are both valid` but space between `[]` and `{` is needed